operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ Entries should be ordered as follows:
 Entries should include a reference to the Pull Request that introduced the change.
 
 ## main / unreleased
+* [ENHANCEMENT] Update memcached statefulset manifest #2321
```

Comment:
please move  to top of the other ENHACEMENTs

Reply:
you mean 1 enhancement per bullet point?

Reply:
ah no ok got it. nvm. i push the update